### PR TITLE
Offboarding

### PR DESCRIPTION
**Pull request by emmabckstrm**
Of course should be merged into master but doing it like this makes it really easy to see which commits are actually pushed in this branch.

Screenshot 2020-09-03 at 11 36 13
Screenshot 2020-09-03 at 11 36 31

### CODE REVIEW

File: db/migrate/20200902_create_subscription_cancellations.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class CreateSubscriptionCancellations < ActiveRecord::Migration[6.0]
```
Comment (emmabckstrm) [inline db/migrate/20200902_create_subscription_cancellations.rb:1]: I wrote this file manually. I'm not sure about the name of the file, the others seem to have more numbers after the date, but couldn't figure out what they were so leaving it like this and asking what to do :D

File: app/models/subscription_cancellation.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class SubscriptionCancellation < ApplicationRecord
```
Comment (stefanlindbohm) [inline app/models/subscription_cancellation.rb:3]: I feel this is not really really representing the cancellation itself (that is in Stripe). What about `SubscriptionCancellationFeedback`?

File: db/migrate/20200902_create_subscription_cancellations.rb (hunk 1 above)
Comment (stefanlindbohm) [reply db/migrate/20200902_create_subscription_cancellations.rb:1]: I think it's just the full timestamp. Best is to generate these files with `bin/rails generate migration CreateSubscriptionCancellation`

File: db/migrate/20200902_create_subscription_cancellations.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+  def change
+    create_table :subscription_cancellations do |t|
+      t.timestamps
+
+      t.timestamp :signed_up
```
Comment (stefanlindbohm) [inline db/migrate/20200902_create_subscription_cancellations.rb:6]: To be in line with other timestamp fields, I propose this gets named `signed_up_at`.

File: app/controllers/users/subscriptions_controller.rb [hunk 4]
```diff
@@ -30,9 +30,16 @@ def update
     end
 
     def destroy
+      cancellation = SubscriptionCancellation.new(
+        signed_up: current_user.created_at,
```
Comment (stefanlindbohm) [inline app/controllers/users/subscriptions_controller.rb:34]: In cases where someone resubscribed, this won't be correct. But you can get to the actual signup date for the current subscription if you move `SubscriptionManager` initialization up and getting it through `@manager.subscription.start_date`.

**Review by stefanlindbohm — COMMENTED** [review]
Looking good!

File: db/migrate/20200902_create_subscription_cancellations.rb (hunk 3 above)
Comment (emmabckstrm) [reply db/migrate/20200902_create_subscription_cancellations.rb]: I gave you one better even, with `subscribed_at`

**Pull request merged**